storage/innobase/srv/srv0start.cc

```diff
@@ -1569,10 +1569,16 @@ static dberr_t recreate_redo_files(lsn_t &flushed_lsn) {
   tables space in case the shutdown wasn't slow. In such case we should start
   from an lsn at least equal to checkpoint_lsn as pages in the tablespace will
   have lsns larger than flushed_lsn. */
+
+#ifdef XTRABACKUP
+  /* PXB recreats redo log file for rollbackup transactionts */
```

Comment:
recreates redo log file for rollback of transactions

Reply:
Done